components/match2/wikis/rocketleague/match_summary.lua

```diff
@@ -41,15 +42,20 @@ local Casters = Class.new(
 
 function Casters:addCaster(caster)
 	if Logic.isNotEmpty(caster) then
-		table.insert(self.casters, '[[' .. caster .. ']]')
+		local nameDisplay = '[[' .. caster.name .. '|' .. caster.dname .. ']]'
+		if caster.flag then
+			table.insert(self.casters, Flags.Icon(caster['flag']) .. ' ' .. nameDisplay)
+		else
+			table.insert(self.casters, nameDisplay)
+		end
 	end
 	return self
 end
 
 function Casters:create()
 	return self.root
-		:wikitext('<b>Caster' .. (#self.casters > 1 and 's' or '') .. ':</b><br>')
-		:wikitext(table.concat(self.casters, ', '))
+		:wikitext('Caster' .. (#self.casters > 1 and 's' or '') .. ': ')
+		:wikitext(table.concat(self.casters, #self.casters > 2 and ', ' or ' & '))
```

Comment:
```suggestion
		:wikitext(mw.text.listToText(self.casters, ', ', ' & '))
```

Reply:
This suggestion produces `Foo, Bar & Baz` rather than `Foo, Bar, Baz`, which reads more descriptively and lacks an Oxford comma, so I prefer it this way. The current code is how it was originally showcased (see start of PR)

Reply:
if RL wants it that way fine for me